apps/console/src/features/users/pages/user-edit.tsx

```diff
@@ -112,7 +113,7 @@ const UserEditPage = (): ReactElement => {
         }
 
         setReadOnlyUserStoresLoading(true);
-        SharedUserStoreUtils.getReadOnlyUserStores().then((response) => {
+        SharedUserStoreUtils.getReadOnlyUserStores().then((response: string[]) => {
```

Comment:
Shall we fix this formatting issue?
```
        SharedUserStoreUtils.getReadOnlyUserStores()
            .then((response: string[]) => {
                setReadOnlyUserStoresList(response);
            })
            .finally(() => {
```

Reply:
Fixed with - https://github.com/wso2/identity-apps/pull/3629/commits/deacda2fec42006c12dbf3c8c53c1fa9908ba4f7